test/e2e/integration/app-frontend/group.ts

```diff
@@ -494,4 +495,42 @@ describe('Group', () => {
       });
     cy.get(appFrontend.group.mainGroup).find(mui.tableElement).should('have.length', 0);
   });
+
+  it.only('Filling out part of the group data in different pages with openByDefault true and first', () => {
```

Comment:
I thought I had made some git hooks to prevent you from even committing `it.only`.. 🤔 Strange! But anyway, this disables all other tests, so it should not go into production..

Reply:
nicely spotted 😅